src/OrchardCore.Modules/OrchardCore.AutoSetup/AutoSetupMiddleware.cs

```diff
@@ -131,13 +119,13 @@ public async Task InvokeAsync(HttpContext httpContext)
                 if (!settings.IsUninitialized())
                 {
                     await _shellHost.ReloadShellContextAsync(_shellSettings, eventSource: false);
-                    httpContext.Response.Redirect(pathBase);
+                    httpContext.Response.StatusCode = 409;
```

Comment:
4xx status codes indicate that the client messed up something, what's not the case here. Why not [503 Service Unavailable](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503)?

Reply:
I thought you would comment here :)